web/packages/teleport/src/DesktopSession/DesktopSession.tsx

```diff
@@ -97,6 +90,16 @@ export function DesktopSession(props: State) {
     screen: 'processing',
     canvasState: { shouldConnect: false, shouldDisplay: false },
   });
+  const keyboardHandler = useRef(new KeyboardHandler());
+
+  useEffect(() => {
+    keyboardHandler.current = new KeyboardHandler();
```

Comment:
Why is this necessary? Do we need to overwrite the keyboard handler we created on line 93?

Reply:
Ah, I blindly copied it from `useTdpClientCanvas` and didn't notice it.
I removed setting the handler again.